INSTALL.md

```diff
@@ -0,0 +1,21 @@
+### Install and run
+
+### Dependencies
+* Node.js, version or above. Download it from https://nodejs.org/en/download/ .
+* Expo CLI. Install it using `npm install -g expo-cli` in your terminal.
+* Optionally, `sharp-cli`. Install it using `npm install -g sharp-cli`.
```

Comment:
@heftymouse what is `sharp-cli` for?

Reply:
@devcer `sharp-cli` was for the production build, Expo gave me a warning saying that it's recommended to use that instead of node for images. I'll remove it if required.

Reply:
Yes. please remove it. I'm not sure if we need it.